pkgs/tools/system/hw-probe/default.nix

```diff
@@ -0,0 +1,108 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, makeWrapper
+
+# Required
+, hwinfo
+, dmidecode
+, smartmontools
+, pciutils
+, usbutils
+, edid-decode
+
+# Recommended
+, withRecommended ? false # Install recommended tools
```

Comment:
I left this as `false` because I noticed that a similar flag is false for inxi: https://github.com/NixOS/nixpkgs/blob/590fa80026dbaa004db6ea68d0697fff2ac4711d/pkgs/tools/system/inxi/default.nix#L3

Not sure what best practice is

Reply:
If you use the tool normally would you always want those programs? If yes, it should be true.

Reply:
Gotcha, I'll fix this